server/lib/subscriptions.js

```diff
@@ -334,6 +343,19 @@ export async function sendThankYouEmail(order, transaction) {
   const relatedCollectives = await order.collective.getRelatedCollectives(3, 0);
   const recommendedCollectives = await getRecommendedCollectives(order.collective, 3);
   const user = order.createdByUser;
+  const orderPlan = getOrderPlan(order);
```

Comment:
You should be more precise is variable names and function name, so anyone can understand easily what's being fetched/checked.